Modules/RaidManager/RaidManager.lua

```diff
@@ -282,6 +283,14 @@ function RaidManager:StartRaid(raid)
         end
     end
 
+    -- Auto reward on-time bonus if set
+    if raid:Configuration():Get("raidCompletionBonus") then
```

Comment:
checking wrong config for on-time bonus: `raidCompletionBonus`

Reply:
Now I understand why the one at raid completion was wrong at first.. changed the wrong one when adding raid end